arangod/Views/LoggerView.h

```diff
@@ -26,6 +26,7 @@
 
```

Comment:
this is outside the scope of the task #344

Reply:
@vasiliy-arangodb The fact of presence LoggerView requires a useless changes in LoggerView.h and LoggerView.cpp to compile ArangoDB successfully which means that it's in the scope of the task. Since LoggerView is no longer needed please remove LoggerView.* files instead of changing them.

Reply:
this is outside the scope of this pull-request